src/coreclr/vm/precode_portable.cpp

```diff
@@ -8,24 +8,24 @@
 #include "precode_portable.hpp"
 
 #ifdef HOST_64BIT
-    #define CANARY_VALUE 0x1234567812345678
+#define CANARY_VALUE 0x1234567812345678
 #else // HOST_64BIT
-    #define CANARY_VALUE 0x12345678
+#define CANARY_VALUE 0x12345678
 #endif // HOST_64BIT
 
 bool PortableEntryPoint::IsNativeEntryPoint(TADDR addr)
```

Comment:
```suggestion
bool PortableEntryPoint::HasNativeEntryPoint(TADDR addr)
```
This can return true for interpreted methods with native call stub, so `Has` makes more sense.

Reply:
Good point.